CharacterCreation/03_Feats.txt

```diff
@@ -200,6 +200,12 @@ head out and looking at your enemy. And somehow you still hit them (some of the
 Roll a d10 before any Blind Firing or Hip Firing action. If the roll is from 6 to 10, 
 Gain -2 miss chance for that firing action; If the roll is from 1 to 5, nothing happens.
 
+Last Stand:
+Prerequisite: Level 4
+Don't die alone. Always bring a friend. You may take an action to do one of 
+the following: Draw a one-handed weapon such as a pistol or SMG, Aim, or fire.
+You can no longer take any of these actions if you are below -50 health. 

```

Comment:
Per our conversation.
Please clarify the writing:
Do you intend for the feat to allow you to take one action per turn or just one action per Down, period?
Also, you removed the part that explains what triggers this ability.